lib/modules/manager/gradle/__fixtures__/1/libs.versions.toml

```diff
@@ -1,17 +1,21 @@
 [versions]
 detekt = "1.17.0"
-kotest = "4.6.0"
+kotest = { require = "4.6.0" }
 publish-on-central = "0.5.0"
 
 [libraries]
 detekt-formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }
 kotest-assertions-core-jvm = { module = "io.kotest:kotest-assertions-core-jvm", version.ref = "kotest" }
 kotest-runner-junit5 = { module = "io.kotest:kotest-runner-junit5", version.ref = "kotest" }
 mockito = { group = "org.mockito", name = "mockito-core", version = "3.10.0" }
+japicmp = { module = "com.github.siom79.japicmp:japicmp", version.require = "0.15.+" }
```

Comment:
Is this a valid maven / Gradle version?

Reply:
Yes -- see https://docs.gradle.org/current/userguide/rich_versions.html for the `require` parameter, and https://docs.gradle.org/current/userguide/single_versions.html for the prefix range used (which appears to be processed by the `gradle` versioning scheme)